website/markdown/docs/commands/dependencies.mdx

```diff
@@ -0,0 +1,211 @@
+
+## How to get started
+
+1. List requirements for third-party dependencies in `Dependencies.swift` manifest.
+2. Install dependencies using `tuist dependencies [subcommand]` command.
+3. Link dependencies with project's targets.
+4. Regenerate project.
+
+## `Dependencies.swift`
+
+Requirements for third-party dependencies are defined in `Dependencies.swift` manifest. `Dependencies.swift` need to import the framework `ProjectDescription` which contains all the classes and enums that are available for you to describe your projects.
+
+You can edit `Dependencies.swift` using `tuist edit` command.
+
+Tuist expects `Dependencies.swift` under `Tuist/` directory:
+
+```bash
+AwesomeProject
+	|- App
+	|- AppTests
+	|- Tuist
+		|- Dependencies.swift
+	|- Project.swift
+```
+
+The snippet below shows an example project manifest:
+
+```swift
+import ProjectDescription
+
+let dependencies = Dependencies([
+    .carthage(origin: .github(path: "Alamofire/Alamofire"), requirement: .exact("5.0.4"), platforms: [.macOS])
+])
+```
+
+### Dependencies
+
+A `Dependencies` manifest allows for defining external dependencies for Tuist.
+
+<PropertiesTable
+  properties={[
+    {
+      name: 'Dependencies',
+      description: 'List of dependencies.',
+      type: '[Dependency]',
+      typeLink: '#dependency',
+      optional: false,
+      default: `[]`,
+    },
+  ]}
+/>
+
+### Dependency
+
+Contains the description of external dependency that can by installed using Tuist.
+
+<EnumTable
+  cases={[
+    {
+      case:
+        '.carthage(origin: CarthageOrigin, requirement: CarthageRequirement, platforms: Set<Platform>)',
+      description:
+        'A description of dependency that can by installed using Carthage',
+    },
+  ]}
+/>
+
+### CarthageOrigin
+
+Origin of the Carthage dependency.
+
+<EnumTable
+  cases={[
+    {
+      case: '.github(path: String)',
+      description:
+        'Mimics `github` keyword from `Cartfile`. GitHub repositories (both GitHub.com and GitHub Enterprise).',
+    },
+    {
+      case: '.git(path: String)',
+      description:
+        'Mimics `git` keyword from `Cartfile`. Other Git repositories.',
+    },
+    {
+      case: '.binary(path: String)',
+      description:
+        'Mimics `binary` keyword from `Cartfile`. Dependencies that are only available as compiled binary `.frameworks`.',
+    },
+  ]}
+/>
+
+### CarthageRequirement
+
+Requirement for the Carthage dependency.
+
+<EnumTable
+  cases={[
+    {
+      case: '.exact(Version)',
+      description: 'Mimics `== 1.0` from `Cartfile`.',
+    },
+    {
+      case: '.upToNext(Version)',
+      description: 'Mimics `~> 1.0` from `Cartfile`.',
+    },
+    {
+      case: '.atLeast(Version)',
+      description: 'Mimics `>= 1.0` from `Cartfile`.',
+    },
+    {
+      case: '.branch(String)',
+      description: 'Mimics `"branch"` from `Cartfile`.',
+    },
+    {
+      case: '.revision(String)',
+      description: 'Mimics `"revision"` from `Cartfile`.',
```

Comment:
To follow up on the comment above - not all users are always intimate with what `revision` is, so it'd be great if they could find in the place where they actually needed instead of being pointed to somewhere else